src/main/scala/com/gd/twitteranalytics/TweetsDataLoader.scala

```diff
@@ -0,0 +1,11 @@
+package com.gd.twitteranalytics
+
+import org.apache.spark.sql.DataFrame
+
+object TweetsDataLoader {
+
+  def saveOutputToHdfs(inputDataFrame:DataFrame,savePath:String,partitionCol:String)={
```

Comment:
is `partitionCol` dynamic / environment dependent ?

Reply:
i.e. does it really make sense to configure it on the function call?

Reply:
Okay, So initial thought process, when I made partitionCol dynamic and put in conf file was that when we will create a DataFrame of will save data, a partitionColumn can be anything. Not just event_date for this particular application for the reuse of functions.
But now if we have removed partitionColumn from conf file, it should be removed from arguments also.